client/core/trade.go

```diff
@@ -600,12 +628,22 @@ func (t *trackedTrade) isRefundable(match *matchTracker) bool {
 	}
 
 	// Issue a refund if our swap's locktime has expired.
-	swapLocktimeExpired, err := wallet.LocktimeExpired(proof.Script)
+	swapLocktimeExpired, contractExpiry, err := wallet.LocktimeExpired(proof.Script)
 	if err != nil {
 		log.Errorf("error checking if locktime has expired for %s contract on order %s, match %s: %v",
 			dbMatch.Side, t.ID(), match.id, err)
 		return false
 	}
+	if !swapLocktimeExpired {
+		swapCoinID := proof.TakerSwap
+		if dbMatch.Side == order.Maker {
+			swapCoinID = proof.MakerSwap
+		}
+		from := t.wallets.fromAsset
+		log.Infof("Contract for match %v with swap coin %v (%s) has an expiry time of %v (%v), not yet expired.",
```

Comment:
Won't this print every `tick` for up to 48 hours?

Reply:
Yeah.  Note sur how to handle that better, except maybe hacking in a once per hour limit per refundable trade, but having it tell you when the refund is ready is desirable.

Reply:
Maybe I'll put a `contractExpiry` field in `trackedTrade` that gets stored the first time it gets computed here.  That way it only need be logged the first time it is determined.

Reply:
Thinking about this... while logging this every tick will be too noisy, logging it just once might defeat the purpose, if the logged message gets eaten up by newer log messages.

I like the idea of printing this message once per hour (or every 30 minutes), something like `if (contractExpiry - now) mod 1 hour <= tickInterval` could work.

Reply:
I made changes similar to what you suggested @itswisdomagain, although it's still necessary to flag if it has been displayed yet so that it can be displayed on the first tick regardless of where that falls in the hour.  Also, you still end up with two of the messages because you have two calls to isRefundable when the negotiation first starts (one via `handleMatchRoute`  and one with the usual `tick` from the ticker in `listen`).  There may be other times it's logged multiple times in that window, but it's fine.  Will push the commit shortly.

Reply:
Now in https://github.com/decred/dcrdex/pull/574/commits/fbd490bf081ccc486fb0e4496a47e5cc2f5a4de2, although it's a but more complex than I'd usually like just for some logging.  I'm OK with it though since it logs the first check and subsequent checks if it's in the first tick window of the hour (with reference to the time until expiry, not wall time).